src/main/kotlin/no/nav/syfo/domain/JournalpostMetadata.kt

```diff
@@ -0,0 +1,10 @@
+package no.nav.syfo.domain
+
+data class JournalpostMetadata(
+    val bruker: Bruker?
+)
```

Comment:
Kan du få en bruker som ikke har `id | type`?

Reply:
Ser du kaster feil om de mangler senere. Burde det tryne allerede når du opprettet brukerobjektet isåfall?

Reply:
Vurderte å gjøre det, men ble lat.. Skjerpings for min del :P Skal fikse det.

Reply:
👮